### ISSUE TITLE
Reply to highlighted text

### ISSUE THREAD

**Issue by tomsmith8** [stakwork/sphinx-mac#206]
**Draft**

if you highlight part of a text msg and hit “reply” just that snipped shows up instead of the whole long message.

Sometimes if its a large comment its hard to know exactly what people are replying to

the ability to reply to the same msg with multiple specific msgs that target a snippet

**Comment by tomastiminskas** [comment]
@tomsmith8 It's kind of hard to implement this right now. Basically we might need changes on relay to send a reply snippet (I would say start character index and length) along with the replyUUID reference when sending the reply. That way the receiver will get the message, the replyUUID (referencing the message it's replying to) and the reply snippet (referencing the part of the original message highlighted). I will need to think more about this but that's the first approach I have in mind

But as we are working on sphinx v2, maybe better to hold on with this until we have the new model working and implement it there.

**Comment by tomsmith8** [comment]
Yeah, V2 feature in my mind too. Thanks for the response. I will noodle on this idea more too
